### PR TITLE
docs: Multiple rephrases

### PR DESCRIPTION
**Pull request by blasrodri**

### CODE REVIEW

File: content/tokio/tutorial/hello-tokio.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ Most computer programs execute in the same order that it is written. The first
 line executes, then the next, and so on.  With synchronous programming, when a
 program encounters an operation that cannot be completed immediately, it will
 block until the operation completes. For example, establishing a TCP connection
-requires an exchange with a peer over the network takes some amount of time. 
-During this time, the thread is blocked.
+requires an exchange with a peer over the network, which can take a seizable
```
Comment (jmgrosen) [inline content/tokio/tutorial/hello-tokio.md:102]: Perhaps you mean "sizeable" here?

File: content/tokio/tutorial/spawning.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ Then create a new, empty `src/main.rs` and continue.
...
-The first thing our Redis server needs to do is accept inbound TCP sockets. This
-is done with [`tokio::net::TcpListener`][tcpl].
+The first thing our Redis server needs to do is to accept inbound TCP sockets.
+This is done with [`tokio::net::TcpListener`][tcpl].
 
 [[info]]
 | Many of Tokio's types are named the same as their synchronous equivalent in
-| the Rust standard library. When it makes sense, Tokio exposes the same APIs
+| the Rust standard library. When feasible, Tokio exposes the same APIs
 | as `std` but using `async fn`.
```
Comment (Darksonn) [inline content/tokio/tutorial/spawning.md:25]: I'm not convinced about this change.

Comment (blasrodri) [reply content/tokio/tutorial/spawning.md:25]: What about: When reasonable?

Comment (Darksonn) [reply content/tokio/tutorial/spawning.md:25]: I prefer the original phrasing, to be honest.

File: content/tokio/tutorial/hello-tokio.md [hunk 3]
```diff
@@ -189,15 +189,16 @@ The return value of an `async fn` is an anonymous type that implements the
...
+found in most of Rust's crates.
 
 1. It is an `async fn`
 2. It is annotated with `#[tokio::main]`
 
 An `async fn` is used as we want to enter an asynchronous context. However,
 asynchronous functions must be executed by a [runtime]. The runtime contains the
 asynchronous task scheduler, provides evented I/O, timers, etc. The runtime does
-not automatically start, so the main function needs to start it.
+not automatically start, thus the main function needs to start it.
```
Comment (hawkw) [inline content/tokio/tutorial/hello-tokio.md:201]: I think the use of "thus" here is actually less correct than "so".

Comment (blasrodri) [reply content/tokio/tutorial/hello-tokio.md:201]: @hawkw you are right!
https://jakubmarian.com/so-thus-therefore-and-hence-in-english/

**Comment by Darksonn** [comment]
Thank you for taking the time to make a PR!

**Pull request merged**